app.rb

```diff
@@ -1,8 +1,19 @@
 require 'sinatra/base'
+require './lib/peep'
+require './database_connection_setup'
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+
+  get '/messages' do
+    Peep.toggle_order if params['sorting']
```

Comment:
This is a clever solution but I'm wondering if it's a little dangerous that the order can be toggled and then left in the toggled state.  What if you were to use Peep.all in another context?

Reply:
An alternative would be to have an optional second parameter on Peep.all which could specify the ordering.  Or, another one, would be to just always return the Peeps in reverse chronological order (I don't think the requirements ask for any flexibility there).